JavaScript/JavaScriptSDK/Context/IApplication.ts

```diff
@@ -0,0 +1,16 @@
+﻿module Microsoft.ApplicationInsights.Context {
+
+    "use strict";
+
+    export interface IApplication {
+        /**
+         * The application version.
+         */
+        ver: string;
```

Comment:
cannot change public API (unless we have a really good reason), it is already exposed to customers